README.md

```diff
@@ -1,16 +1,55 @@
 # Redis shopping cart demo
 
-Check each README.md in client or server folders.
+This shopping cart is using Redis and RedisJson module functionalities, allowing you to save JSON as keys using methods like json_get and json_set
 
 ![alt text](https://github.com/RemoteCraftsmen/redis-shopping-cart/blob/main/preview.png?raw=true)
 
+## Hot to run it locally?
+
+### Prerequisites
+
+- Node - v12.19.0
+- NPM - v6.14.8
+- Docker - v19.03.13 (optional)
+
+### Installation
+
+```
+# Environmental variables
+
```

Comment:
Go to `server` folder and then:

Reply:
done (a bit differently so with client is the same)